mod_reforged/hooks/skills/effects/stunned_effect.nut

```diff
@@ -0,0 +1,56 @@
+::mods_hookExactClass("skills/effects/stunned_effect", function(o) {
+	o.m.DefaultColor <- ::createColorWhite();
+	o.m.DoubleStunColor <- ::createColor("#ff6600");	// Orange
+
+	local setTurns = o.setTurns;
+	o.setTurns = function( _turns )
+	{
+		setTurns(_turns);
+		this.updateSprite();
+	}
+
+	o.onUpdate = function( _properties )
```

Comment:
Is a complete overwrite of `onUpdate` really necessary? Maybe you let the original `onUpdate` run first and then just append `this.updateSprite()` to it? And in `updateSprite` you check if the sprite is visible, and if yes, then update color, otherwise return.

Reply:
I adjusted that now. Overwriting was not necessary.